### PR TITLE
Update journal-of-medical-internet-research.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/comment/276685/#Comment_276685
Add DOI
Add webcite function (via extra field)
remove et al
add ISBN

### CODE REVIEW

File: journal-of-medical-internet-research.csl [hunk 1]
```diff
@@ -218,6 +238,16 @@
       </if>
     </choose>
   </macro>
+  <macro name="webcite">
+    <choose>
+      <if type="webpage post-weblog" match="any">
+        <group delimiter=": " prefix=". ">
+          <text value="archived at" text-case="capitalize-first"/>
+          <text variable="note"/>
```
Comment (adam3smith) [inline journal-of-medical-internet-research.csl]: I'm skeptical about this. This is one journal; we can't really expect users to change their data entry patterns for this and on the negative side this is going to add all sorts of incomprehensible clutter to the bibliography. I'd say comment this out

Comment (damnation333) [inline journal-of-medical-internet-research.csl]: Is there any other way of solving this? Can users manually somehow add the webcite link then? usually people don't cite many website anyway, so it wouldn't be the majority of entries anyway.

Comment (adam3smith) [inline journal-of-medical-internet-research.csl]: I'd say just add it as part of the final edit. I don't see how we'd reasonably automate this, no. We could use the archive field, but that's not available for webpages, so of little use there.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**